index.html

```diff
@@ -0,0 +1,67 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8" />
+  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+  <title>Document</title>
+  <link rel="preconnect" href="https://fonts.googleapis.com" />
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+  <link href="https://fonts.googleapis.com/css2?family=Crete+Round&display=swap" rel="stylesheet" />
+  <link rel="stylesheet" href="style.css" />
+</head>
+
+<body>
+  <!-- Header  & Navigation Panel -->
+  <header>
+    <nav>
+      <section class="logo"><a href="#">My Logo</a></section>
+      <section class="nav_bar_contents">
+        <ul id="nbc">
+          <li><img src="images/Portoflio.png" alt="" /></li>
+          <li><img src="images/Contact.png" alt="" /></li>
+          <li><img src="images/About.png" alt=""></li>
+          <li><img src="images/Icon - Mail.png" alt="" /></li>
+        </ul>
+      </section>
+      <section class="menu"><img src="images/Union.png" alt="" /></section>
+    </nav>
+  </header>
+
+  <!-- Main Content -->
+  <section class="body-2">
+    <div class="txt">
+      <div class="h1">
+        <p>
+          Hey There.<br>I’m Nahin.
+        </p>
+      </div>
+      <div class="h2">
+        <h2>I'm a Software Developer</h2>
+      </div>
+      <div class="h3">
+        <p>
+          I can help you build a product , feature or website Look through
+          some of my work and experience! If you like what you see and have
+          a project you need coded, don’t hesitate to contact me.
+        </p>
+      </div>
+    </div>
+    
+      <ul id="sm">
```

Comment:
- [ ] Kindly put the list of icons using a `ul` tag to group them, and `li` per icon; each icon has a link (`a` tag).

* Apart from being semantically correct, the `a` tag will let you connect your social networks with the icons, using href.